src/utils.jl

```diff
@@ -288,3 +261,58 @@ function range_shuffle!(r::UnitRange, a::AbstractVector; seed::Int=-1)
     end
 end
 
+"""
+    randbn(n, p, seed=-1)
+
```

Comment:
This was added from some other module. It's nothing to do with graphs so it belongs here.